IFC4x3/Sections/Resource definition data schemas/Schemas/IfcRepresentationResource/Entities/IfcGeographicCRS/DocEntity.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeographicCRS" UniqueId="2d6be663-8d5e-42cc-b5b4-13bb37f41262" BaseDefinition="IfcCoordinateReferenceSystem" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="GeodeticDatum" UniqueId="b4d581f7-612a-4e98-963a-95ad06714397" DefinedType="IfcIdentifier" AttributeFlags="1" />
+		<DocAttribute Name="Unit" UniqueId="82bf71dc-4f74-483b-84c7-70c84a5ce22d" DefinedType="IfcNamedUnit" AttributeFlags="1" />
```

Comment:
Proposal: I imagine this entity as:

![grafik](https://user-images.githubusercontent.com/59165496/148699627-c73c0a81-cee1-4c2b-8665-4774d5ed0b7a.png)

Reply:
Is `PrimeMeridian` still a thing? https://github.com/pjanck/Georef-IFC4x3-rework/issues/28

Reply:
Yes, similarly to how `IfcProjectedCRS` has attributes for `MapZone` etc. this is the parallel in geographic CRS.